share/revng/clang-format-style-file.yml

```diff
@@ -115,7 +115,7 @@
   PenaltyBreakTemplateDeclaration: 10,
   PenaltyExcessCharacter: 751362,
   PenaltyIndentedWhitespace: 0,
-  PenaltyReturnTypeOnItsOwnLine: 0,
+  PenaltyReturnTypeOnItsOwnLine: 21,
```

Comment:
I've no strong opinions on this.  Both before and after look good to me.

How did you come up with 21?

Reply:
Binary search :). I noticed that llvm uses 60 and the behaviour is different from ours (0), so I tried values until I found the breaking point. It seems to be `>20` - hence 21.

Reply:
Scary but OK.

Reply:
I'd assume this `20` is some value hardcoded somewhere in `clang-format` sources, but I never got a chance to look for it :)